namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-frontend-staging/resources/route53.tf

```diff
@@ -0,0 +1,34 @@
+resource "aws_route53_zone" "cla_frontend_route53_zone" {
+  name = "cases.civillegaladvice.service.gov.uk"
+
+  tags = {
+    team_name              = var.team_name
+    business-unit          = var.business-unit
+    application            = var.application
+    is-production          = var.is-production
+    environment-name       = var.environment-name
+    infrastructure-support = var.infrastructure-support
+  }
+}
+
+resource "kubernetes_secret" "cla_frontend_route53_zone_sec" {
+  metadata {
+    name      = "cla-frontend-route53-zone-output"
+    namespace = var.namespace
+  }
+
+  data = {
+    zone_id = aws_route53_zone.cla_frontend_route53_zone.zone_id
+  }
+}
+
+resource "aws_route53_record" "add_a_record" {
+  name    = "cases.civillegaladvice.service.gov.uk"
+  zone_id = aws_route53_zone.cla_frontend_route53_zone.zone_id
+  type    = "A"
+  alias {
+    name                   = "dualstack.cla-front-elbprodf-1o815cnz2w3lh-1554019512.eu-west-1.elb.amazonaws.com."
+    zone_id                = "Z24D9N37DW2MHW"
```

Comment:
I think it is aws_route53_zone. cla_frontend_route53_zone not the parent

Reply:
Hi Raz not too sure what you mean here. Do you mean to replace the hard coded zone id with the dynamic one?

Reply:
Ok I just saw your comment on slack. The hard coded zone id here is from template deploy

Reply:
yes, so  aws_route53_zone. cla_frontend_route53_zone, we'll put the NS records in the parent in DSD after creation

Reply:
Hey Raz just wanting to get a better understanding. What's NS? And what has this got to do with DSD? Do we need to do more work outside of what I've done here?